ui/src/pages/Tenant/New/TenantSource.tsx

```diff
@@ -1,25 +1,24 @@
 import { getAllTenants } from '@/services/tenant';
 import { intl } from '@/utils/intl';
 import { useRequest } from 'ahooks';
-import type { RangePickerProps } from 'antd/es/date-picker';
-import type { Dayjs } from 'dayjs';
-import moment from 'moment';
-import dayjs from 'dayjs';
 import {
-  Card,
-  Col,
-  DatePicker,
-  Form,
-  Input,
-  Row,
-  Select,
-  Space,
-  Switch,
-  TimePicker,
-  Tooltip,
+Card,
```

Comment:
Why do you remove leading spaces here?

Reply:
The internationalization tool automatically formats.

Reply:
But I can modify the configuration of the tool so that it formats the code in the ideal way